dlt/destinations/dataset.py

```diff
@@ -42,18 +90,91 @@ def __init__(
         self.iter_arrow = self._wrap_iter("iter_arrow")  # type: ignore
         self.iter_fetch = self._wrap_iter("iter_fetch")  # type: ignore
 
+    # TODO: where should this go, should cursors support "native" ibis or do we do a conversion somewhere
+    def ibis(self, chunk_size: int = None) -> Optional[IbisTable]:
```

Comment:
this does not make sense... we have stuff materialized. way better to just do df() which has data frame interface

Reply:
I removed those